vars/containerPushToRegistry.groovy

```diff
@@ -101,7 +102,7 @@ void call(Map parameters = [:]) {
                 ) {
                     sourceBuildImage.pull()
                 }
-                sh "docker tag ${config.sourceRegistry}/${config.sourceImage} ${config.dockerImage}"
+                sh "docker tag ${q(config.sourceRegistry)}/${q(config.sourceImage)} ${q(config.dockerImage)}"
```

Comment:
Wouldn't this work and still be secure? This would avoid the inner quotes.
```suggestion
                sh "docker tag ${q(config.sourceRegistry+'/'+config.sourceImage)} ${q(config.dockerImage)}"
```

Reply:
Yes of course. In this case it is easy. But why are inner quotes an issue?

Reply:
It can confuse developers reading the tests. They might not understand why, until they read the productive code.
But I agree, we have several other cases like this where we should **not** change it and continue quoting each parameter.